### PR TITLE
Column Customization: Fix up settings button

### PR DESCRIPTION
**Pull request by JamesHollyer**
* Motivation for features / changes
Fixes up the button in settings which opens the editor. This change makes it an actual button with an edit icon. It also adds some functionality to highlight the button when the editor is open.

### CODE REVIEW

File: tensorboard/webapp/metrics/views/right_pane/settings_view_component.scss [hunk 1]
```diff
@@ -101,9 +101,16 @@ mat-slider {
   align-items: center;
   display: flex;
   cursor: pointer;
+  font-size: inherit;
+  height: 25px;
```
Comment (bmd3k) [inline tensorboard/webapp/metrics/views/right_pane/settings_view_component.scss:105]: can we do height: 28px?

File: tensorboard/webapp/metrics/views/right_pane/settings_view_component.scss [hunk 2]
```diff
@@ -101,9 +101,16 @@ mat-slider {
...
+  padding-left: 5px;
+  padding-right: 8px;
```
Comment (bmd3k) [inline tensorboard/webapp/metrics/views/right_pane/settings_view_component.scss:107]: can we do padding-right: 12px?

File: tensorboard/webapp/metrics/views/right_pane/settings_view_component.scss [hunk 3]
```diff
@@ -101,9 +101,16 @@ mat-slider {
...
+  &.toggle-opened {
+    background-color: mat.get-color-from-palette(mat.$grey-palette, 800);
+  }
```
Comment (bmd3k) [inline tensorboard/webapp/metrics/views/right_pane/settings_view_component.scss:110]: Looking at the Settings button, which also has an icon to align, it sets the following on .mat-button-wrapper, which seems to work well:

display: inline-flex;
align-items: center;

File: tensorboard/webapp/metrics/views/right_pane/settings_view_component.scss [hunk 4]
```diff
@@ -101,9 +101,16 @@ mat-slider {
   align-items: center;
   display: flex;
   cursor: pointer;
+  font-size: inherit;
+  height: 25px;
+  padding-left: 5px;
+  padding-right: 8px;
+  &.toggle-opened {
+    background-color: mat.get-color-from-palette(mat.$grey-palette, 800);
```
Comment (bmd3k) [inline tensorboard/webapp/metrics/views/right_pane/settings_view_component.scss:109]: Note that this doesn't work well in light mode.

**Review by bmd3k — APPROVED** [review]

File: tensorboard/webapp/metrics/views/right_pane/settings_view_component.scss (hunk 4 above)
Comment (JamesHollyer) [reply tensorboard/webapp/metrics/views/right_pane/settings_view_component.scss]: Oops. Fixed that up.

File: tensorboard/webapp/metrics/views/right_pane/settings_view_component.scss (hunk 3 above)
Comment (JamesHollyer) [reply tensorboard/webapp/metrics/views/right_pane/settings_view_component.scss:117]: That works great!

**Review by JamesHollyer — COMMENTED** [review]
Good suggestions.

**Pull request merged**